frontend/src/components/ui/KoFiButton.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Box, Button, Typography } from '@mui/material';
+
+export default function KoFiButton({
+  color = "primary.main",
+  label = "Buy us a coffee",
+  borderRadius = "12px"
+}) {
+  return (
+    <Box sx={{ display: 'flex' }}>
+      <Button
+        component="a"
+        href={`https://ko-fi.com/asianweddingmakeup`}
```

Comment:
do we want to redirect users to ko-fi or use their floating widget so users don't leave our site?

https://help.ko-fi.com/hc/en-us/articles/360018381678-Ko-fi-donation-widget#h_01HZ6B9S97S93K9KBBNF3PFV4H

Reply:
The widget is a floating button that's always at the bottom of your window, which looks annoying visually. I personally hate when windows get crowded with floating things. But if there's a way to get a normal button, I'm happy to switch! I'll get the change out now, and we can follow up later